src/DynamoCore/Library/FunctionDescriptor.cs

```diff
@@ -172,7 +172,7 @@ public FunctionDescriptor(FunctionDescriptorParams funcDescParams)
 
         public string Summary
         {
-            get { return summary ?? (summary = this.GetSummary(pathManager)); }
+            get { return summary ?? (summary = this.GetSummary()); }
```

Comment:
I removed `PathManager` from `TypedParameter`. But I can't remove it from `FunctionDescriptor` since it's used to get xml documentation file.